packages/evolution-common/src/services/surveyGeneration/SurveyGenerator.ts

```diff
@@ -0,0 +1,21 @@
+import { SurveyWidgetConfig } from "../../../../../example/demo_survey/src/config/widgets.config";
```

Comment:
The function should rather be generic, it cannot depend on a file from the application itself